### PR TITLE
Align update codes (almost) for all scripts

### PR DESCRIPTION
**Pull request by rdlrt**

### CODE REVIEW

File: scripts/cnode-helper-scripts/topologyUpdater.sh [hunk 1]
```diff
@@ -51,24 +51,38 @@ shift $((OPTIND -1))
...
+  if [[ $(command -v checkUpdate) ]]; then
+    echo -e "\nCould not find checkUpdate function in env, make sure you're using official guild docos for installation!"
+    myExit 1
+  fi
+  # check for env update
+  ! checkUpdate env ${BATCH_AUTO_UPDATE} && myExit 1
+  ! checkUpdate topologyUpdater.sh ${BATCH_AUTO_UPDATE} && myExit 1
+  ! . "${PARENT}"/env && myExit 1 "ERROR: Topology Updater failed to load common env file\nPlease verify set values in 'User Variables' section in env file or log an issue on GitHub"
+  # source common env variables in case it was updated
+  . "${PARENT}"/env
```
Comment (Scitz0) [inline scripts/cnode-helper-scripts/topologyUpdater.sh:79]: Double source:ing of env. Remove the first one and also make it offline, no need to source in online mode for TU.

File: scripts/cnode-helper-scripts/topologyUpdater.sh [hunk 2]
```diff
@@ -51,24 +51,38 @@ shift $((OPTIND -1))
...
-  *) exit 1 ;;
-esac
+. "${PARENT}"/env offline &>/dev/null # ignore any errors, re-sourced later
+
+if [[ "${UPDATE_CHECK}" == "Y" ]] && [[ "${BATCH_AUTO_UPDATE}" == "N" ]]; then
+  echo "Checking for script updates..."
+  # Check availability of checkUpdate function
+  if [[ $(command -v checkUpdate) ]]; then
+    echo -e "\nCould not find checkUpdate function in env, make sure you're using official guild docos for installation!"
+    myExit 1
```
Comment (Scitz0) [inline scripts/cnode-helper-scripts/topologyUpdater.sh:72]: See previous comment for myExit

**Review by Scitz0 — COMMENTED** [review]
Added a couple of comments.

File: scripts/cnode-helper-scripts/cncli.sh [hunk 3]
```diff
@@ -120,25 +120,36 @@ getLedgerData() { # getNodeMetrics expected to have been already run
...
-  # Check if update is available
-  ! checkUpdate env ${BATCH_AUTO_UPDATE} && exit 1
-  ! checkUpdate cncli.sh ${BATCH_AUTO_UPDATE} && exit 1
   
+  . "${PARENT}"/env offline &>/dev/null # ignore any errors, re-sourced later
+  
+  if [[ "${UPDATE_CHECK}" == "Y" ]]; then
+    echo "Checking for script updates..."
+    # Check availability of checkUpdate function
+    if [[ $(command -v checkUpdate) ]]; then
```
[the remark this answers is not in the document]
Comment (rdlrt) [reply scripts/cnode-helper-scripts/cncli.sh:144]: discussed offline, need tty for condition to pass, agreed to keep as-is

File: scripts/cnode-helper-scripts/cncli.sh [hunk 4]
```diff
@@ -120,25 +120,36 @@ getLedgerData() { # getNodeMetrics expected to have been already run
...
+  #######################################################
+  # Version Check                                       #
+  #######################################################
+  clear
+  
   if [[ ! -f "${PARENT}"/env ]]; then
     echo -e "\nCommon env file missing: ${PARENT}/env"
     echo -e "This is a mandatory prerequisite, please install with prereqs.sh or manually download from GitHub\n"
-    exit 1
+    myExit 1
```
[the remark this answers is not in the document]
Comment (rdlrt) [reply scripts/cnode-helper-scripts/cncli.sh:136]: Adding, thanks

File: scripts/cnode-helper-scripts/cntools.sh [hunk 5]
```diff
@@ -64,10 +69,24 @@ fi
...
+    # check for env update
+    ! checkUpdate env && myExit 1
+    ! . "${PARENT}"/env && myExit 1 "ERROR: CNTools failed to load common env file\nPlease verify set values in 'User Variables' section in env file or log an issue on GitHub"
+    # source common env variables in case it was updated
+    . "${PARENT}"/env
+    case $? in
+      1) myExit 1 ;;
+      2) clear ;;
+    esac
+  fi
```
[the remark this answers is not in the document]
Comment (rdlrt) [reply scripts/cnode-helper-scripts/cntools.sh]: Good catch

**Review by Scitz0 — APPROVED** [review]
👍

**Pull request merged**